fastlane/Fastfile

```diff
@@ -57,6 +57,12 @@ ENV['PUBLIC_CONFIG_FILE'] = PUBLIC_CONFIG_FILE
 INTERNAL_CONFIG_FILE = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.internal.xcconfig')
 ENV['FASTLANE_WWDR_USE_HTTP1_AND_RETRIES'] = 'true'
 
+# Fastlane's `git_branch` action and it's relevant helpers use environment variables to modify the output.
+# That means if we change the branch as part of an action, it'll return the incorrect branch.
+# This environment variable disables that behavior.
+# See https://github.com/fastlane/fastlane/pull/21597
+ENV['FL_GIT_BRANCH_DONT_USE_ENV_VARS'] = true
```

Comment:
Nitpick but also a question. I thought environment variables can only be strings - or not set at all. I guess Ruby might be setting this as a string, I am not sure - but for those who are less familiar with Ruby (like myself) I think it's worth setting this as a string value instead.

Having said that, I might totally be wrong about environment variables, and if so, I'd like to fix that 😅

Reply:
<img width="977" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1218433/9f798eeb-6fc6-4953-90f2-de6b62c1b70b">

Nah, you are totally right. I just assumed it was fine and that Ruby handled the conversion under the hood. My mistake. Thanks for noticing 🙇‍♂️